sdk/python/feast/feature_store_config.py

```diff
@@ -23,9 +23,9 @@ class Config:
 
     def __init__(
         self,
-        provider: Optional[str],
-        online_store: Optional[str],
-        metadata_store: Optional[str],
+        provider: Optional[str] = None,
```

Comment:
Shouldn't we be defining the default provider here instead of in the constructor?